### ISSUE TITLE
Wrong autodetected image size

### ISSUE THREAD

**Issue by gseric** [endroid/QrCode#67]
Hello!

If image size is not set, it is detected automatically. Auto detection uses module size and the number of modules. That's good idea but it seems that detected size is not optimal so image is scaled/distorted a bit. For example, if there are 33x33 modules and module size is 1, auto detected size is 41x41 instead of 33x33.

I debugged this problem and found that size autodetection works this way:
```php
$mib = $max_modules_1side + 8;
$this->size = $mib * $qrcode_module_size;
```
Seems that cause of this issues is addition of 8. Is it really needed?

Regards

**Comment by gseric** [comment]
I understand now why you added 8 modules (`$mib = $max_modules_1side + 8`) - it means 8 modules of "quiet zone" (4 on each side) - but the problem is that quiet zone modules are **also added when quiet zone is disabled**.

**Comment by gseric** [comment]
I hope I fixed this issue in #68
